lib/__tests__/cli.test.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
```

Comment:
I think these tests are not needed, because they test `meow` package and not stylelint code. `meow` package has own set of tests, we don't need to write and support tests for it.

Reply:
Ideally, I agree. But we don't have now system test cases to verify all CLI options.

https://github.com/stylelint/stylelint/blob/d3c4a9f50bb77fd9014e04b84e9b336978239d97/system-tests/cli/cli.test.js#L11

I'd like just to avoid regressions caused by a big refactoring for type-checking. That is, the regression is whether our `meow()` call output is broken or not.
I think easier to write unit test cases instead of system test cases for this purpose.